models/chunter/src/index.ts

```diff
@@ -44,6 +48,9 @@ export class TComment extends TAttachedDoc implements Comment {
   @Prop(TypeMarkup(), chunter.string.Message)
   @Index(IndexKind.FullText)
   message!: string
+
+  @Prop(Collection(attachment.class.Attachment), attachment.string.Attachments)
```

Comment:
I supposed attachment will be for document comment is attachedTo, and comment will just reference to attachment, instead